### ISSUE TITLE
Send sms

### ISSUE THREAD

**Issue by myannr** [Mobideck/appdeck#49]
Hi, 
I try to send sms with appDeck but when I use 
sms:
in a link, appDeck try to open another page as a web link, s it does never call the sms function of the phone.

Is there a way to do it ?

Regards

**Comment by rwanito** [comment]
Hi !
Try to use the share API. [Share API on Appdeck website](http://www.appdeck.mobi/doc/api_share)
User needs to select SMS app to send an sms. I hope that answered your questions.

**Comment by myannr** [comment]
Hi, thank you for your answer but it's not reaaly what I need.
with the sms: function, the number of the receiver is ever selected so, the user can directly send the sms to him.

with the Share API, it's the content that is ever entered, but I do not know what the user wants to say to the receiver.

Regards

**Comment by mdekermadec** [comment]
Hello, the fact that appdeck does open sms: correctly is a bug.

maybe a workarround would be to use app.loadextern("sms:");

loadextern allow you to open any URL by operating system.

**Comment by myannr** [comment]
Hi, I try this function but it change nothing on Android.
So I am still blocked.

**Comment by RoboKopf** [comment]
Yes, app.loadextern() works on iOS but not Android. It has been like this early February at least.

**Comment by mdekermadec** [comment]
Last version add a new Service API: http://www.appdeck.mobi/doc/api_service

You can use it to send SMS/Email and open Web Browser.

AppDeck app on AppStore/PlayStore/Online Emulator should be updated within next few days.

**Comment by RoboKopf** [comment]
Thank you for this new Service API! Is it better to do (1) or (2) below to open a page in the browser? Or are they equivalent?

(1) app.loadextern('http://www.google.fr/')

(2) app.service.openLink('http://www.google.fr/')

Is app.loadextern deprecated now?
